### ISSUE TITLE
[Bug] Free Storage space not shown

### ISSUE THREAD

**Issue by heyiamsven** [MauriceNino/dashdot#111]
### Description of the bug

Hello again,

I wanted to report that dashdot is unable to show my free storage space in the pie chart. I have this issue since I started using dashdot, so it wasn't introduced with the last updates.

Image of the issue:
https://abload.de/img/storage_dashdotjsj1m.png

Im starting dashdot with:
> docker container run -d -it \
>   -p 3001:3001 \
>   -v /etc/os-release:/etc/os-release:ro \
>   -v /proc/1/ns/net:/mnt/host_ns_net:ro \
>   --privileged \
>   --env DASHDOT_OVERRIDE_STORAGE_SIZE_1="137438953472" \
>   --env DASHDOT_OVERRIDE_STORAGE_BRAND_1="SanDisk" \
>   --env DASHDOT_OVERRIDE_STORAGE_TYPE_1="MicroSD" \
>   --env DASHDOT_OVERRIDE_RAM_TYPE="DDR4" \
>   --env DASHDOT_OVERRIDE_RAM_FREQUENCY="2133" \
>   --env DASHDOT_OVERRIDE_CPU_BRAND="ARM" \
>   --env DASHDOT_OVERRIDE_CPU_MODEL="S922X" \
>   --env DASHDOT_OVERRIDE_OS="armbian" \
>   --env DASHDOT_OVERRIDE_RAM_BRAND="Hardkernel" \
>   --env DASHDOT_ENABLE_CPU_TEMPS="true" \
>   --env DASHDOT_NETWORK_WIDGET_ENABLE="false" \
>   --name dashdot \
>   mauricenino/dashdot

### Relevant log output

```shell
```

### What browsers are you seeing the problem on?

Chrome

### Where is your instance running?

Other (Please specify below)

### Additional context

Odroid N2+

**Comment by MauriceNino** [comment]
Hey thanks for the issue. Can you try checking `df` on your host and mount the mountpoints in the container? (e.g. `-v /mnt:/host_mnt`). You might need to switch to the dev image for it to work (`mauricenino/dashdot:dev`) - I will release it soon.

**Comment by MauriceNino** [comment]
Or nvm I see you are running on a single drive. Can you please execute the following command and paste the output?

```
docker container exec -it CONTAINER_NAME /bin/sh -c 'apk add git && git clone https://github.com/MauriceNino/disk-info /disk-info && cd /disk-info && npm i && node .'
```

**Comment by heyiamsven** [comment]
Output:
> root@odroidn2:/# docker container exec -it dashdot /bin/sh -c 'apk add git && git clone https://github.com/MauriceNino/disk-info /disk-info2 && cd /disk-info2 && npm i && node .'
> OK: 81 MiB in 67 packages
> Cloning into '/disk-info2'...
> remote: Enumerating objects: 6, done.
> remote: Counting objects: 100% (6/6), done.
> remote: Compressing objects: 100% (5/5), done.
> remote: Total 6 (delta 0), reused 6 (delta 0), pack-reused 0
> Receiving objects: 100% (6/6), done.
> 
> added 1 package, and audited 2 packages in 873ms
> 
> 1 package is looking for funding
>   run `npm fund` for details
> 
> found 0 vulnerabilities
> npm notice
> npm notice New minor version of npm available! 8.11.0 -> 8.12.1
> npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.12.1
> npm notice Run npm install -g npm@8.12.1 to update!
> npm notice
> Disks []
> Sizes [
>   {
>     fs: 'overlay',
>     type: 'overlay',
>     size: 124272218112,
>     used: 7897223168,
>     available: 115075604480,
>     use: 6.42,
>     mount: '/'
>   },
>   {
>     fs: '/dev/mmcblk0p1',
>     type: 'ext4',
>     size: 124272218112,
>     used: 7897223168,
>     available: 115075604480,
>     use: 6.42,
>     mount: '/etc/os-release'
>   }
> ]

**Comment by MauriceNino** [comment]
Please run the following command and paste the output again:

```
docker container exec -it dashdot /bin/sh -c "export LC_ALL=C; lsblk -ablJO 2>/dev/null; unset LC_ALL"
```

**Comment by heyiamsven** [comment]
> root@odroidn2:~# docker container exec -it dashdot /bin/sh -c "export LC_ALL=C; lsblk -ablJO 2>/dev/null; unset LC_ALL"
> {
>    "blockdevices": [
>       {
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop0",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:0",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop0",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop0",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop1",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:1",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop1",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop1",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop2",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:2",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop2",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop2",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop3",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:3",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop3",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop3",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop4",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:4",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop4",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop4",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop5",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:5",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop5",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop5",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop6",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:6",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop6",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop6",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 0,
>          "disc-max": 0,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "loop7",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "7:7",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "loop7",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/loop7",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": true,
>          "rq-size": 256,
>          "sched": "mq-deadline",
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "loop",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 4194304,
>          "disc-max": 3741319168,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": true,
>          "kname": "mmcblk0",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "179:0",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "mmcblk0",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/mmcblk0",
>          "phy-sec": 512,
>          "pkname": null,
>          "pttype": "dos",
>          "ptuuid": "cbf654cd",
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": false,
>          "rq-size": 64,
>          "sched": "none",
>          "serial": "0xaceb3869",
>          "size": 127865454592,
>          "start": null,
>          "state": null,
>          "subsystems": "block:mmc:mmc_host:platform",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "disk",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 4194304,
>          "disc-max": 3741319168,
>          "disc-zero": false,
>          "fsavail": "115075133440",
>          "fsroots": [
>              "/var/lib/docker/containers/c50a3aaf37ee17b0399defe18bc701a1fdcf732d9e7e806850a535e94db845b3/hosts", "/var/lib/docker/containers/c50a3aaf37ee17b0399defe18bc701a1fdcf732d9e7e806850a535e94db845b3/hostname", "/var/lib/docker/containers/c50a3aaf37ee17b0399defe18bc701a1fdcf732d9e7e806850a535e94db845b3/resolv.conf", "/etc/os-release"
>          ],
>          "fssize": "124272218112",
>          "fstype": "ext4",
>          "fsused": "7897694208",
>          "fsuse%": "6%",
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": true,
>          "kname": "mmcblk0p1",
>          "label": null,
>          "log-sec": 512,
>          "maj:min": "179:1",
>          "min-io": 512,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "mmcblk0p1",
>          "opt-io": 0,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": "0x83",
>          "parttypename": null,
>          "partuuid": "cbf654cd-01",
>          "path": "/dev/mmcblk0p1",
>          "phy-sec": 512,
>          "pkname": "mmcblk0",
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": false,
>          "rq-size": 64,
>          "sched": "none",
>          "serial": null,
>          "size": 126579900416,
>          "start": 8192,
>          "state": null,
>          "subsystems": "block:mmc:mmc_host:platform",
>          "mountpoint": "/etc/hosts",
>          "mountpoints": [
>              "/etc/hosts", "/etc/hostname", "/etc/resolv.conf", "/etc/os-release"
>          ],
>          "tran": null,
>          "type": "part",
>          "uuid": "9576c5ad-92ed-47f0-91aa-c3d36e5bcf7a",
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 4096,
>          "disc-max": 2199023255040,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": "swap",
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "zram0",
>          "label": null,
>          "log-sec": 4096,
>          "maj:min": "252:0",
>          "min-io": 4096,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "zram0",
>          "opt-io": 4096,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/zram0",
>          "phy-sec": 4096,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": false,
>          "rq-size": 128,
>          "sched": null,
>          "serial": null,
>          "size": 1983881216,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": "[SWAP]",
>          "mountpoints": [
>              "[SWAP]"
>          ],
>          "tran": null,
>          "type": "disk",
>          "uuid": "c03b8014-9d51-428a-9e5a-0771de77ee5c",
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 4096,
>          "disc-max": 2199023255040,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": "ext4",
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "zram1",
>          "label": "log2ram",
>          "log-sec": 4096,
>          "maj:min": "252:1",
>          "min-io": 4096,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "zram1",
>          "opt-io": 4096,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/zram1",
>          "phy-sec": 4096,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": false,
>          "rq-size": 128,
>          "sched": null,
>          "serial": null,
>          "size": 52428800,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "disk",
>          "uuid": "fb5b2d00-9262-4e29-ba4f-7ef91ba29afa",
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       },{
>          "alignment": 0,
>          "disc-aln": 0,
>          "dax": false,
>          "disc-gran": 4096,
>          "disc-max": 2199023255040,
>          "disc-zero": false,
>          "fsavail": null,
>          "fsroots": [
>              null
>          ],
>          "fssize": null,
>          "fstype": null,
>          "fsused": null,
>          "fsuse%": null,
>          "fsver": null,
>          "group": "disk",
>          "hctl": null,
>          "hotplug": false,
>          "kname": "zram2",
>          "label": null,
>          "log-sec": 4096,
>          "maj:min": "252:2",
>          "min-io": 4096,
>          "mode": "brw-rw----",
>          "model": null,
>          "name": "zram2",
>          "opt-io": 4096,
>          "owner": "root",
>          "partflags": null,
>          "partlabel": null,
>          "parttype": null,
>          "parttypename": null,
>          "partuuid": null,
>          "path": "/dev/zram2",
>          "phy-sec": 4096,
>          "pkname": null,
>          "pttype": null,
>          "ptuuid": null,
>          "ra": 128,
>          "rand": false,
>          "rev": null,
>          "rm": false,
>          "ro": false,
>          "rota": false,
>          "rq-size": 128,
>          "sched": null,
>          "serial": null,
>          "size": 0,
>          "start": null,
>          "state": null,
>          "subsystems": "block",
>          "mountpoint": null,
>          "mountpoints": [
>              null
>          ],
>          "tran": null,
>          "type": "disk",
>          "uuid": null,
>          "vendor": null,
>          "wsame": 0,
>          "wwn": null,
>          "zoned": "none",
>          "zone-sz": 0,
>          "zone-wgran": null,
>          "zone-app": 0,
>          "zone-nr": 0,
>          "zone-omax": 0,
>          "zone-amax": 0
>       }
>    ]
> }
> 
>

**Comment by MauriceNino** [comment]
Ok I have created an issue in the repository which might have caused the error, which is fixed now. Can you please pull the latest dev image (`mauricenino/dashdot:dev`) and see if the problem is fixed? If not, please run `docker exec CONTAINER yarn cli raw-data --storage` and paste the output.

**Comment by heyiamsven** [comment]
When pulling the latest dev image I receive the following output:
> WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
> 7b3886ae729413518035e31012a9145c21b19febe55d191617cec2f19c61264e

When running `docker exec dashdot yarn cli raw-data --storage` I get the following output:
> docker exec dashdot yarn cli raw-data --storage
> yarn run v1.22.19
> error Command "cli" not found.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>

**Comment by MauriceNino** [comment]
ah damn, yeah, dev images only work on amd64. I will just release a main version, as there are other changes waiting to be released as well.

**Comment by MauriceNino** [comment]
I have released a new main version (please switch back to `mauricenino/dashdot:latest` and re-run the commands from my previous comment).

**Comment by heyiamsven** [comment]
Yep, now it is working as it should. Thankyou so much again! <3

![fixed](https://user-images.githubusercontent.com/18595670/173350414-54a20ae6-3f66-42bb-8fd2-28d062d84886.PNG)

**Comment by MauriceNino** [comment]
Ok, the new main release has introduced some new errors, so I reverted this bug fix for now on the main release. I will keep it open, but you can stay on that version for now if it works for you.

If the main issue gets resolved (https://github.com/sebhildebrandt/systeminformation/issues/695), I will notify you here, so you can update again.

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 3.3.3

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/CHANGELOG.md) for more details.

**Comment by heyiamsven** [comment]
> @heyiamsven Can you confirm that the issue is resolved on the latest release?

Issue is resolved on the latest release!
